src/components/Autocomplete/README.md

```diff
@@ -80,6 +81,14 @@ class AutocompleteExample extends React.Component {
     this.filterAndUpdateOptions(newValue);
   };
 
+  clearText = () => {
+    const {inputText} = this.state;
+    if (inputText == '') {
```

Comment:
Strict equality check would be preferred over a loose equality check here however we normally go with this shorthand 

```suggestion
    if (!inputText) {
```

Reply:
My bad, sorry. It should also reset the `filterAndUpdateOptions`. @darrenhebner I don't want to hijack your PR. Do you want me to make these changes or do it yourself?

```suggestion
   clearText = () => {
    const {inputText} = this.state;
    if (!inputText) {
      return;
    }
    this.setState({inputText: ''});
    this.filterAndUpdateOptions('');
  };
```

Reply:
@solonaarmstrong - Feel free to make changes to the README 👍